packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -286,6 +286,7 @@ export enum FieldMetadataType {
   Currency = 'CURRENCY',
   Date = 'DATE',
   DateTime = 'DATE_TIME',
+  Domain = 'DOMAIN',
```

Comment:
Let's rename this to LINKS and try to do something that accommodate different type of links with the options, including domain. We could have two field type DOMAINS and LINKS but it's probably not worth it

Reply:
Done!